libraries/chain/include/steem/chain/account_object.hpp

```diff
@@ -124,6 +126,10 @@ namespace steem { namespace chain {
          }
 
          asset effective_vesting_shares()const { return vesting_shares - delegated_vesting_shares + received_vesting_shares; }
+#ifdef STEEM_ENABLE_SMT
+         /// Return SMT token object controlled by this account identified by its symbol. May return nullptr!
+         const smt_token_object* get_controlled_smt( const asset_symbol_type& smt_symbol, database& db ) const;
```

Comment:
This appears to be beyond the scope of this PR.

Reply:
What is the problem here ? To avoid duplication of code referencing the SMT in each place we have  added method to the account.

Reply:
No problem with the code.

Please see my comment above. It is difficult to review a PR when there code for other issues mixed in.